bskyembed/src/locale/locales/en/messages.ts

```diff
@@ -0,0 +1,4 @@
+/*eslint-disable*/ import type {Messages} from '@lingui/core'
```

Comment:
Maybe it should be added to `.gitignore` list just like [this](https://github.com/bluesky-social/social-app/blob/5020e4233a5a50be8fe7b8f360737da4d30ab74a/.gitignore#L112).